qualtran/bloqs/mcmt/and_bloq.py

```diff
@@ -160,7 +160,7 @@ def add_my_tensors(
 
     def wire_symbol(self, soq: 'Soquet') -> 'WireSymbol':
         if soq.reg.name == 'target':
-            return directional_text_box('∧', side=soq.reg.side)
+            return directional_text_box('&', side=soq.reg.side)
```

Comment:
motivate this change? can qpic not do unicode? document this restriction somewhere in the wire symbol world. (The previous symbol was the unicode character for "LOGICAL AND")

Reply:
reverted